documentation/Demo-Server-Guide.md

```diff
@@ -269,11 +268,27 @@ From there the following commands might be useful:
 
 1. Log in to the server using terminal
 2. Copy the log to an accessible place (in the server) i.e. if getting log for instance 5000:
-`sudo cp /var/lib/docker/volumes/conforma-on-8000_logs/_data/server.log demo_server/2022-11-15.log`
-3. Now exit the server `exit` and on your local machine open the terminal 
+   `sudo cp /var/lib/docker/volumes/conforma-on-8000_logs/_data/server.log demo_server/2022-11-15.log`
+3. Now exit the server `exit` and on your local machine open the terminal
 4. After navigate to a folder where you would like to save the log, run (some similat to this):
-\```
-scp -i $KEY_LOC ubuntu@conforma-demo.msupply.org:/home/ubuntu/demo_server/2022-11-15.log .
-\```
+   \```
+   scp -i $KEY_LOC ubuntu@conforma-demo.msupply.org:/home/ubuntu/demo_server/2022-11-15.log .
+   \```
+
 - KEY_LOC is an environment variable for the server public key :)
-- And the `.` means it will create a file with the same file name on the current folder
\ No newline at end of file
+- And the `.` means it will create a file with the same file name on the current folder
+
+## Managing CloudVPS hosted servers
+
+Conforma live servers are usually hosted on ([CloudVPS](https://www.cloudvps.com/)), an ([openstack](https://www.openstack.org/)) server provider.
+
+### Chaning CloudVPS server settings
+
+Prior to changing any settings on CloudVPS, it is a good idea to take a backup snapshot of the live server, and ensure back ups are logging correctly to dropbox or wherever you are keeping your backups.
+
+1. Log in to ([CloudVPS](https://openstack.cloudvps.com/auth/login/?next=/)). Credentials are available through Bitwarden.
+2. Access all instances through compute / instances
```

Comment:
Typo "compute" ?

Reply:
This isn't a typo, but maybe just weird language from the CloudVPS client. 

See:
https://www.cloudvps.com/knowledgebase/entry/2679-openstack-getting-started/

Reply:
Okay cool